Common/dirs.cpp

```diff
@@ -55,10 +56,7 @@ string Dirs::tempDir()
 	if (p != "")
 		return p;
 
-#ifdef _WIN32
-	//lets just read it from the environment if the anwser is found there
-	if(ProcessInfo::inWinContainer()) {
-		Log::log() << "In sandbox win container!" << std::endl;
+	if(localAppDataDir() == "") { //we are an engine
 		char* buf = nullptr;
 		size_t sz = 0;
 		if(_dupenv_s(&buf, &sz, "JASP_TMP_DIR") == 0 && buf != nullptr) {
```

Comment:
_dupenv_s is a microsoft function.
This cannot be compiled on Mac (and probably on Linux).